app/src/main/java/com/xa/xpensauditor/ShowTransActivity.java

```diff
@@ -18,6 +18,9 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First sentence of Javadoc is missing an ending period.